src/components/MDXComponents/MDXCode.tsx

```diff
@@ -30,6 +30,18 @@ const addVersions = (code: string) => {
   return code;
 };
 
+const hasHighlights = (code: string): boolean => {
+  const highlightableStrings = [
+    '// highlight-start',
+    '// highlight-end',
+    '// highlight-line-start'
```

Comment:
Do we need to also add `// highlight-next-line` ?
```suggestion
    '// highlight-start',
    '// highlight-end',
    '// highlight-line-start',
    '// highlight-next-line'
```